lca-cli/installation_configuration_files/scripts/prepare-installation-configuration.sh

```diff
@@ -68,3 +68,9 @@ systemctl disable prepare-installation-configuration.service
 if [[ -n "${DEVICE}" ]]; then
     umount_config "${DEVICE}"
 fi
+
+if [ "${LCA_TEST_inject_failure_config:-no}" = "yes" ]; then
```

Comment:
All this is done for test? 
In anycase i am going to remove this file in nearest future so not sure this is a good way to do it.
Code will be moved to post-pivot